src/troute-nwm/src/nwm_routing/__main__.py

```diff
@@ -1034,11 +2039,16 @@ async def main_v03_async(argv):
         help="Use version 2 or 3 of the input format. Default 3",
     )
     v_args = v_parser.parse_known_args()
-    if v_args[0].input_version == 4:
-        LOG.info("Running main v03 - async looping")
-        coroutine = main_v03_async(v_args[1])
-        asyncio.run(coroutine)
+    #if v_args[0].input_version == 4:
+    #    LOG.info("Running main v03 - async looping")
+    #    coroutine = main_v03_async(v_args[1])
+    #    asyncio.run(coroutine)
         # loop.run_until_complete(coroutine)
+
     if v_args[0].input_version == 3:
         LOG.info("Running main v03 - looping")
         main_v03(v_args[1])
+    if v_args[0].input_version == 4:
```

Comment:
Again, let's try to get to a single main function.  If backwards compatibility is important, we can discuss ways to support that, but with 4 versions of main and an async v3 interface, it is reasonably confusing.